client/components/password-change/password-change.js

```diff
@@ -0,0 +1,39 @@
+import "./index.css";
+import React, {useState} from "react";
+
+const PasswordChange = () => {
+  const [oldPassword, changeOldPassword] = useState();
+  const [newPassword, changeNewPassword] = useState();
+
+  const handleSubmit = (oldPassword, newPassword) =>
+    console.log({oldPassword, newPassword});
+
+  return (
+    <div className="owisp-passchange-container">
+      <div className="title">Change your password</div>
+      <input
+        style={{marginTop: "1rem"}}
+        type="password"
+        name="oldPassword"
+        placeholder="Old Password"
+        onChange={e => changeOldPassword(e.target.value)}
+      />
+      <input
+        type="password"
+        name="newPassword"
+        placeholder="New Password"
+        onChange={e => changeNewPassword(e.target.value)}
+      />
+      <div
+        className="owisp-passchange-submit"
+        role="button"
+        onClick={() => handleSubmit(oldPassword, newPassword)}
+        onKeyDown={() => handleSubmit(oldPassword, newPassword)}
+      >
+        Submit
+      </div>
+    </div>
+  );
+};
+
+export default PasswordChange;
```

Comment:
Also, I think there is no need to write this in a separate component. Include it to the PasswordChangePage component and rename that to PasswordChange.